src/cavendish_particle_tracks/_decay_angles_dialog.py

```diff
@@ -135,11 +187,19 @@ def _setup_decayangles_layer(self):
     def _on_click_calculate(self) -> None:
         """When 'Calculate' button is clicked, calculate opening angles and populate table"""
 
+        # The Lambda travels towards the decay vertex, line needs to be reversed
+        lambda_line = self.cal_layer.data[0][::-1]
+        proton_line = self.cal_layer.data[1]
+        pion_line = self.cal_layer.data[2]
+
+        # Save the lines
+        self.alines = [lambda_line, proton_line, pion_line]
+
```

Comment:
Why are these changes included?

Reply:
Because the function that connects the decay vertex of the Λ and the origin vertices of the proton and π is much simpler if the vertex index is the same for all the `Line` objects. Hence, the Λ's direction needs to be reverted for the angle calculation.

Reply:
Ah. Ok, fair enough.